packages/mobile/src/navigator/Navigator.tsx

```diff
@@ -594,23 +597,46 @@ export function MainStackScreen() {
       hasSeenVerificationNux,
     } = mapStateToProps(store.getState())
 
+    // Remove Onboarding Education Screen Experiment: Because remote configs are fetched after the initial route is launched,
+    // The randomization is hardcoded by device id here to achieve a 50/50 split, the value is written into the redux store so the same experience
+    // would persist.
+    // e2e tests cannot mock value with jest thus need to default to true
+    // This block of code should be removed when the experiment is done.
+    const _shouldSkipOnboardingEducationScreen = isE2EEnv ? true : getRandomByUUID() < 0.5
+
     let initialRoute: InitialRouteName
 
     if (!language) {
       initialRoute = Screens.Language
     } else if (!name || !acceptedTerms || pincodeType === PincodeType.Unset) {
       // User didn't go far enough in onboarding, start again from education
-      initialRoute = Screens.OnboardingEducationScreen
-    } else if (!account) {
-      initialRoute = choseToRestoreAccount
-        ? Screens.ImportWallet
+
+      ValoraAnalytics.track(
+        _shouldSkipOnboardingEducationScreen
+          ? OnboardingEvents.onboarding_education_skipped
+          : OnboardingEvents.onboarding_education_not_skipped
+      )
+      initialRoute = _shouldSkipOnboardingEducationScreen
+        ? Screens.Welcome
         : Screens.OnboardingEducationScreen
+    } else if (!account) {
+      if (choseToRestoreAccount) {
+        initialRoute = Screens.ImportWallet
+      } else {
+        ValoraAnalytics.track(
```

Comment:
nit: these ternary expressions are duplicated; maybe worth pulling them out somehow? similar story with `initialRoute`, but don't want to sacrifice clarity for compactness.